e2e/runner/cypress-runner-run-tests.js

```diff
@@ -12,7 +12,7 @@ const isFolder = !!folder;
 const isOpenMode = args["--open"];
 
 const getSourceFolder = folder => {
-  return `./e2e/test/scenarios/${folder}/**/*.cy.spec.js`;
+  return `./e2e/test/scenarios/${folder}/**/*.cy.spec.{js,ts}`;
```

Comment:
is there a separate PR in master you can target allowing .ts specs? If not, could you pull this out as a separate thing?

Reply:
oops thanks, I had already done that and forgot to rebase

Reply:
Here’s the PR that was merged: https://github.com/metabase/metabase/pull/36813